config/import-control.xml

```diff
@@ -240,6 +240,13 @@
     <allow pkg="com.puppycrawl.tools.checkstyle.xpath.iterators" local-only="true"/>
     <allow class="com.puppycrawl.tools.checkstyle.TreeWalkerAuditEvent" local-only="true"/>
 
+    <file name="Abstract.*Node" regex="true">
+      <allow class="com.puppycrawl.tools.checkstyle.api.AbstractSyntaxTree"/>
```

Comment:
remove this old line.
also please make `<file name="Abstract(Element|Root)Node" regex="true">`

allowance should be strict to catch any new classes.

Reply:
Removed old line.

The purpose of `.*` was to catch any future nodes added and not be specific to specific classes. We are already limiting this to the xpath package. I can change `.*` to `\w` but let me know if you still want this fully removed.

Reply:
Ok